README.md

```diff
@@ -128,16 +109,3 @@ When a bug or feature is identified which requires a change.
 10. Change request for anything big (in accordance with ITIL principles)
 11. Deploy/update component in 'prod' environment
 
-**Writing and running [Ansible](Ansible/README.md) playbooks is the recommended practice for deployment of applications.**
-
-Tips for Rspec Tests:
```

Comment:
This is heavily catered to testing with Fedora so I guess yeah feel free to remove as not all that relevant anymore. But we do still use rspec (PushmiPullyu and Discovery for example)